plugins/editorplaceholder/plugin.js

```diff
@@ -72,7 +72,12 @@
 			data = editor.getData();
 
 		if ( isFullPage ) {
-			data = data.match( fullPageRegex )[ 1 ];
+			var bodyDataMatched = data.match( fullPageRegex );
+
+			if ( bodyDataMatched && bodyDataMatched.length > 1 ) {
+				//found data inside `<body>` element
```

Comment:
I think this comment is not necessary here, but you may add reference to issue number above `if` like:

```javascript
// Check if body element exists in editor HTML (#4253).
if (...
```

Reply:
Fixed with: https://github.com/ckeditor/ckeditor4/pull/4370/commits/b3f52357deea82a0a3d8fe1c7af125175a39131a